src/Bridge/NelmioApiDoc/Parser/ApiPlatformParser.php

```diff
@@ -236,7 +236,7 @@ private function parseProperty(ResourceMetadata $resourceMetadata, PropertyMetad
                     return $data;
                 }
 
-                $data['subType'] = $subProperty['subType'];
+                $data['subType'] = $subProperty['dataType'];
```

Comment:
@soyuka yep, thats why i ask for howto test impact ;) i just discovered platform v2 2 days ago and try to upgrade a project